### PR TITLE
BZ 1236354

### PR DESCRIPTION
**Pull request by bfallonf**
As per BZ suggestion, I switched around the two "Creating from image stream" methods:

https://bugzilla.redhat.com/show_bug.cgi?id=1236354

### CODE REVIEW

File: admin_guide/install/first_steps.adoc [hunk 1]
```diff
@@ -64,21 +64,21 @@ link:../../using_images/db_images/mysql.html[*MySQL*], and
...
 ----
 $ oc create -f \
-    examples/image-streams/image-streams-centos7.json \
+    examples/image-streams/image-streams-rhel7.json \
     -n openshift
 ----
 
-Alternatively, if you are using node hosts that are subscribed using Red Hat
-Subscription Management and prefer to use the Red Hat Enterprise Linux (RHEL) 7
+Alternatively, to create the core set of image streams when using the CentOS 7
```
Comment (adellape) [inline admin_guide/install/first_steps.adoc]: The "when using" here is a little strange/misleading. The original ", which use CentOS 7 based images" was more accurate.

Comment (bfallonf) [inline admin_guide/install/first_steps.adoc]: @adellape Cool. Changed to "...image streams that use the Centos..."

'Cos that vs which.

**Comment by adellape** [comment]
:+1:

**Comment by bfallonf** [comment]
:turtle:

**Pull request merged**